weather/xamarin/Weather/Weather.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Weather.Forms</AssemblyName>
+    <UserSecretsId>8834abd2-d94c-4850-8f31-75d07e4518ff</UserSecretsId>
```

Comment:
Is this intentional?

Reply:
Good question. What is this for?

Reply:
I have no idea where that came from!
Think VS2019 generated it when I checking out the menu option.